gr-blocks/grc/blocks.tree.yml

```diff
@@ -114,9 +107,6 @@
 - Modulators:
   - blocks_vco_f
   - blocks_vco_c
-- Networking Tools:
-  - blocks_tuntap_pdu
-  - blocks_socket_pdu
```

Comment:
I agree that these PDU blocks should go into `gr-pdu` for now. Though, I'd expect a `gr-network` or `gr-ip` module that groups all the network stuff together at some point.

Reply:
These two blocks are now in gr-network.